controllers/users.js

```diff
@@ -28,9 +27,9 @@ router.post('/', function(req, res) {
 // Update a user
```

Comment:
Yeah as of now they can add any field, but we should definitely force them to match the model attributes eventually haha